libs/sdk-ui-ext/src/internal/tests/mocks/referencePointMocks.ts

```diff
@@ -1946,6 +1946,137 @@ export const multipleMetricsNoCategoriesReferencePoint: IReferencePoint = {
     },
 };
 
+export const onlyMeasuresReferencePoint: IReferencePoint = {
```

Comment:
There is already `oneMetricNoCategoriesReferencePoint` similar ref point. I would change it to `noMeasureInEitherBucketRefPoint` or `noMeasureInAnyBucketRefPoint` or something similar so you know just from the name that it has two measure buckets. Same applies for other reference point mocks that has two measure buckets.